### PR TITLE
Add commands /hide and /show

### PR DESCRIPTION
**Pull request by DeathlyPlays**
This allows Global Staff to say ``/hide`` which will make them appear as regular user on the userlist, often helpful when you are AFK.

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1971,6 +1971,37 @@ exports.commands = {
...
+				if (Config.groupsranking.indexOf(target) <= Config.groupsranking.indexOf(user.group)) {
+					tar = target;
+				} else {
+					this.sendReply('The group symbol you have tried to use is of a higher authority than you have access to. Defaulting to \'' + tar + 'instead.');
+				}
+			} else {
+				this.sendReply('You are now hiding your auth symbol as \'' + tar + '\'.');
+			}
+		}
+		user.getIdentity = function (roomid) {
```
Comment (Slayer95) [inline chat-plugins/info.js]: No...

Comment (Slayer95) [inline chat-plugins/info.js]: Support for this should go in the ``User`` class

**Comment by Slayer95** [comment]
Policy-wise,last time this was discussed, it was approved for admins only by EeveeGeneral.

**Comment by DeathlyPlays** [comment]
So is it wanted or ?_?

**Comment by Zarel** [comment]
For one thing, this is the wrong name for such a command, for another, I never approved this and at this time I don't think this is a good idea.

File: chat-plugins/info.js [hunk 2]
```diff
@@ -1971,6 +1971,35 @@ exports.commands = {
 		"/htmlbox [message] - Displays a message, parsing HTML code contained.",
 		"!htmlbox [message] - Shows everyone a message, parsing HTML code contained. Requires: ~ & #",
 	],
+
+	hide: 'hideauth',
+	hideauth: function (target, room, user) {
+		if (!this.can('lock')) return false;
+		let tar = ' ';
```
Comment (panpawn) [inline chat-plugins/info.js]: Hardcoding ranks like this isn't a good idea

**Comment by panpawn** [comment]
On another note, since you mentioned this being helpful for when staff go AFK, instead we should really add a proper AFK system.

Is there an issue for that open, @Zarel ? Should there be? I remember you had discussed it before, where it would gray out your username on the userlist and stuff, but I don't remember your specific ideas so much.

**Comment by Zarel** [comment]
Yes, I'd support a proper AFK system.

**Comment by Slayer95** [comment]
I guess I'll keep enjoying my % ¯\_(ツ)_/¯

**Comment by Zarel** [comment]
Have you considered just turning on `/away +` and not being in public rooms?

**Comment by DeathlyPlays** [comment]
That's what this added @Zarel where you can disguise as any rank below yours

**Comment by Zarel** [comment]
And I told you I didn't want that

**Comment by DeathlyPlays** [comment]
Indeed

**Comment by DeathlyPlays** [comment]
The  `/away +` idea kinda made me think you meant this

**Comment by panpawn** [comment]
No, look at how ``/away`` works.

It takes on option target and does other commands

**Comment by Zarel** [comment]
`/away +` means that you're blocking PMs and challenges from anyone whose rank is below `+`

**Comment by DeathlyPlays** [comment]
I see now lol

**Comment by DeathlyPlays** [comment]
Forget I said anything

**Pull request closed without merging**